selftests/unit/utils/test_software_manager.py

```diff
@@ -1,4 +1,5 @@
 import os
+import shutil
```

Comment:
As this test is depending on external artifacts, I think it fits better under `functional`.